src/spacel/cloudformation/elb-service.template

```diff
@@ -218,6 +218,7 @@
     "DnsRecord": {
       "Type": "AWS::Route53::RecordSetGroup",
       "Condition": "HasVirtualHost",
+      "DependsOn": "Asg",
```

Comment:
TLDR handling DNS in the template is still "fast" (i.e. makes for a good hack-day demo :P), but scary for production services.